fcrepo-configs/src/main/resources/config/file-s3/repository.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type": "file",
+            "path" : "${fcrepo.object.directory:target/objects}"
+        },
+        "binaryStorage" : {
+            "type" : "s3",
+            "username" : "${aws.accessKeyId}",
```

Comment:
I know it's not proper JSON, but Modeshape does not appear to care if you include comments e.g.,

```
       "username" : "${fcrepo.postgresql.username}",
        // Please note that passing passwords using command-line parameters...
        "password" : "${fcrepo.postgresql.password}"
    }
```

So it's possible that Modeshape is already pre-processing the JSON, or using a JSON library that tolerates comments.